catalog/MDCCatalog/MDCCatalogStyle.m

```diff
@@ -16,10 +16,22 @@
 
 #import "MDCCatalogStyle.h"
 
+static inline BOOL isRunningiOS9OrGreater() {
+   static NSOperatingSystemVersion iOS9Version = {9, 0, 0};
+   NSProcessInfo *processInfo = [NSProcessInfo processInfo];
+   return [processInfo isOperatingSystemAtLeastVersion:iOS9Version];
+}
+
 @implementation MDCCatalogStyle
 
 + (UIFont *)headerFont {
-  return [UIFont fontWithName:@"RobotoMono-Regular" size:14];
+   if (isRunningiOS9OrGreater()) {
+      return [UIFont monospacedDigitSystemFontOfSize:16 weight:UIFontWeightRegular];
+   } else {
+      UIFont* font = [UIFont systemFontOfSize:16];
+      UIFontDescriptor *descriptor = [[font fontDescriptor] fontDescriptorWithSymbolicTraits:UIFontDescriptorTraitMonoSpace];
```

Comment:
This line exceeds the [100-column limit](http://google.github.io/styleguide/objcguide.html). Please break it up. I believe this would be sufficient (note the 4-space indentation for a continued line):

```objectivec
UIFontDescriptor *descriptor =
    [[font fontDescriptor] fontDescriptorWithSymbolicTraits:UIFontDescriptorTraitMonoSpace];
```